main.go

```diff
@@ -0,0 +1,71 @@
+/**
+ *Author: Stefan
+ *Date: 12/12/2019
+ *Last changes: 12/13/2019 15.05
+ *Task: Web server handling any request to it’s address and returning JSON like:
+ */
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+//Response struct
+type Response struct {
+	Host       string   `json:"host"`
+	UserAgent  string   `json:"user_agent"`
+	RequestURI string   `json:"request_uri"`
+	Headers    struct { //is correctly accoding the tasks
+		Accept    string
+		UserAgent string
+	} `json:"headers"`
+}
+
+func jsonHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
It's better to set it in success case, because in case of an error you return not a json, client can fail to handle response